swabbie-core/src/test/kotlin/com/netflix/spinnaker/swabbie/ResourceTypeHandlerTest.kt

```diff
@@ -578,17 +601,50 @@ object ResourceTypeHandlerTest {
     override fun deleteResources(
       markedResources: List<MarkedResource>,
       workConfiguration: WorkConfiguration
-    ): ReceiveChannel<MarkedResource> = produce {
+    ) {
+      markedResources.forEach { m ->
+        simulatedCandidates
+          ?.removeIf { r -> m.resourceId == r.resourceId }.also {
+            //todo eb: add to task tracking repo?
+            if (it != null && it) {
```

Comment:
This is a strange conditional? `it` is a `Boolean`?

Reply:
yes. Fixed the naming so it's more obvious.